src/rpc/blockchain.cpp

```diff
@@ -1520,6 +1521,7 @@ static RPCHelpMan getmempoolinfo()
                         {RPCResult::Type::NUM, "size", "Current tx count"},
                         {RPCResult::Type::NUM, "bytes", "Sum of all virtual transaction sizes as defined in BIP 141. Differs from actual serialized size because witness data is discounted"},
                         {RPCResult::Type::NUM, "usage", "Total memory usage for the mempool"},
+                        {RPCResult::Type::STR_AMOUNT, "total_fee", "Total fees for the mempool in " + CURRENCY_UNIT + ", ignoring modified fees through prioritizetransaction"},
```

Comment:
I don't think the sum of the modified fees has any meaning. They are used to prioritize individual transactions (and thus their packages) for mining. It is not a fee that can be redeemed in any way and in most cases it will be an arbitrary number.

Though, if they are added they can be called `total_modified_fee`, without breaking any clients. I'll leave as is for now.

Reply:
Yes, makes sense!